File: src/model/DefaultRoute.java
package model;

public class DefaultRoute {
    
    private long id;
    private TrailerType trailerType;
    private boolean extraRoute;

    public DefaultRoute(long id, TrailerType trailerType, boolean extraRoute) {
        this.id = id;
        this.trailerType = trailerType;
        this.extraRoute = extraRoute;
    }

    public DefaultRoute(TrailerType trailerType, boolean extraRoute) {
        this.trailerType = trailerType;
        this.extraRoute = extraRoute;
    }
    
    /**
     * @return the id
     */
    public long getID() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(long id) {
        this.id = id;
    }

    /**
     * @return the trailerType
     */
    public TrailerType getTrailerType() {
        return trailerType;
    }

    /**
     * @return the extraRoute
     */
    public boolean isExtraRoute() {
        return extraRoute;
    }
    

}

File: Model/DefaultRoute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    [DataContract()]
    public class DefaultRoute
    {
        [DataMember()]
        public long ID { get; set; }
        [DataMember()]
        public double TrailerType { get; set; }
        [DataMember()]
        public Boolean ExtraRoute { get; private set; }

        public DefaultRoute(long id, double trailerType, Boolean extraRoute)
        {
            this.ID = id;
            this.TrailerType = trailerType;
            this.ExtraRoute = extraRoute;
        }

        public DefaultRoute(double trailerType, Boolean extraRoute)
        {
            this.TrailerType = trailerType;
            this.ExtraRoute = extraRoute;
        }
    }
}
